pyccl/nl_pt/power.py

```diff
@@ -51,11 +51,22 @@ class PTCalculator(object):
         C_window (float): `C_window` parameter used by FAST-PT
             to smooth the edges and avoid ringing. See FAST-PT
             documentation for more details.
+        k_cutoff (float): exponential cutoff scale. All power
+            spectra will be multiplied by a cutoff factor of the
+            form :math:`\\exp(-(k/k_*)^n)`, where :math:`k_*` is
+            the cutoff scale. This may be useful when using the
```

Comment:
This is useful functionality, but it seems it is more logically placed in the transform part of the code. Why have it here?